### PR TITLE
Add test for adding draft value sets

### PR DESCRIPTION
**Pull request by mayerm94**
This is a PR that adds a test to make sure uploading a draft measure works. It uploads the measure marked as draft, checks that the upload happened successfully and that the value sets were versioned appropriately. This PR also adds items from  https://github.com/projecttacoma/bonnie_bundler/commit/4c35242400a2ba8f93deae1946935b4281c563bc that were mistakenly removed
Pull requests into Bonnie Bundler require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] JIRA ticket for this PR: NA
- [x] JIRA ticket links to this PR NA
- [x] Code diff has been done and been reviewed
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA test links: N/A
- [x] Justification for using JIRA tests: N/A
- [x] JIRA tests have been added to sprint N/A

**Reviewer 1:**

Name: Hossenlopp
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases.
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass N/A - N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A - N/A

**Reviewer 2:** @c-monkey

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass - N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree - N/A

### CODE REVIEW

File: test/unit/load_mat_export_test.rb [hunk 1]
```diff
@@ -7,6 +7,28 @@ class LoadMATExportTest < ActiveSupport::TestCase
...
+    @cql_draft_measure_mat_export = File.new File.join('test', 'fixtures', 'DRAFT_CMS2_CQL.zip')
+  end
+
+  test "Loading a CQL Mat export zip file including draft, with VSAC credentials" do
+    VCR.use_cassette("valid_vsac_response_includes_draft") do
+      dump_db
+      user = User.new
+      user.save
+      measure_details = { 'episode_of_care'=> false }
+      Measures::CqlLoader.load(@cql_draft_measure_mat_export, user, measure_details, ENV['VSAC_USERNAME'], ENV['VSAC_PASSWORD'], overwrite_valuesets=false, cache=false, includeDraft=true, ticket_granting_ticket=nil).save
```
Comment (hossenlopp) [inline test/unit/load_mat_export_test.rb]: Something doesn't seem right about the parameters here that are provided as `parameter=value`.  Wouldn't that just pass those four in as `true`, `true`, `true`, `true`?

**Review by c-monkey — APPROVED** [review]
Matt and I carefully reviewed all changes from Alex's PR and there are no longer any inadvertent changes:
`git diff -w 4c35242400a2ba8f93deae1946935b4281c563bc`

**Review by hossenlopp — APPROVED** [review]
